lib/utils.js

```diff
@@ -137,3 +137,20 @@ exports.noOp = function (_cb) {
 	}
 
 };
+
+/**
+ * Return true if object is empty else false
+ * @param {Object} obj   Object to be tested if it is empty or not
+ * @return {Bollean}
+ */
+exports.isEmptyObject = function (obj) {
```

Comment:
removed the `isEmptyObject` function that was added to utils.js and all its references used in overView UI code. replaced it  with `Object.keys(<obj>).length` to determine if object has any keys.